docs/Projects/LLM-benchmarks/project_description.md

```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
+
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
+a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with
+coding and essay writing. In the context of the government, tasks can include for example supporting FOIA
+requests and aiding in answering questions of citizens.
+
+While the potential benefit of using LLM's is large, there are also significant risks.
+Basically an LLM is just a next token predictor, which bases its predictions on the user input (context) and
+on compressed information seen during training (LLM parameters); hence there
+is no guarantee on the quality and correctness of the output. Moreover, due to
+bias in the training data, LLM's can have bias in their output, despite best efforts
+to mitigate this. Additionally, we have human values that we expect LLM's to be aligned with.
+Certainly, within the context of a government, we should take utmost care not to discriminate.
+To assess the quality, correctness, bias and alignment with human values of a LLM one can perform
+benchmarks.
+
+## The project
+
+The LLM-benchmark project of the AI-Validation Team of the Ministry of the Interior and Kingdom
+Relations of the Government of the Netherlands aims to create a platform where LLM's can be measured
+across a wide range of benchmarks. Both LLM's and the benchmarks can be configured by users of the platform.
```

Comment:
I miss the piece here still what the project is exactly, you state here users can configure them, this does imply that it is some kind of platform where they can be ran.